<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #1e1e1e; /* Dark grey background */
    color: #dcdcdc; /* Light grey text, bold for aggression */
    font-weight: bold;
}
QPushButton {
    background-color: #5364A5; /* Blue background for buttons */
    color: white;
    border-style: solid;
    border-width: 2px;
    border-radius: 0px; /* Sharper corners */
    border-color: #5364A5;
    padding: 6px;
    min-width: 80px;
    font-weight: bold; /* Bolder text */
}
QPushButton:hover {
    background-color: #43528B; /* Darker blue for hover */
    color: #ffffff;
    border-color: #43528B;
}
QPushButton:pressed {
    background-color: #333E66; /* Even darker blue when pressed */
    border-style: inset;
}
QLineEdit, QTextEdit, QPlainTextEdit, QSpinBox, QDoubleSpinBox, QComboBox {
    background-color: #2b2b2b;
    color: #dcdcdc;
    border-radius: 0px; /* Sharper edges */
    padding: 2px;
    border: 1px solid #555;
    font-weight: bold; /* Bolder text */
}
QLabel, QCheckBox, QRadioButton {
    color: #dcdcdc;
    font-weight: bold; /* Bolder text */
}
QMenuBar, QMenu {
    background-color: #2b2b2b;
    color: #dcdcdc;
    border: 1px solid #555;
    font-weight: bold; /* Bolder text for menu */
}
QMenuBar::item:selected, QMenu::item:selected {
    background-color: #5364A5; /* Blue for selected menu items */
    color: white;
}
QScrollBar:vertical {
    border: 1px solid #2b2b2b;
    background: #2b2b2b;
    width: 10px;
    margin: 22px 0 22px 0;
}
QScrollBar::handle:vertical {
    background: #5364A5; /* Blue scrollbar handle */
    min-height: 20px;
}
QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    border: 1px solid #2b2b2b;
    background: #5364A5; /* Blue for scrollbar buttons */
    height: 20px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}
/* TABS */
QTabWidget::pane {
    border-top: 2px solid #5364A5; /* Blue top border for tabs */
}

QTabWidget::tab-bar {
    left: 5px; /* move to the right by 5px */
}

QTabBar::tab {
    background: #2b2b2b;
    color: #dcdcdc;
    border: 1px solid #555;
    border-bottom-color: #5364A5; /* Blue for non-selected tab border */
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    padding: 5px;
    min-width: 80px;
    align-items: center;
    justify-content: center;
    max-height: 20px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: #5364A5; /* Blue for selected/hover tab */
    color: white;
}

QTabBar::tab:selected {
    border-color: #5364A5; /* Blue border for selected tab */
    border-bottom-color: #1e1e1e; /* Same as window background color */
}

QTabBar::tab:!selected {
    margin-top: 2px; /* Make non-selected tabs look smaller */
}

/* Q Tree Widget */
QTreeWidget {
    border: 1px solid #555;
    background-color: #2b2b2b;
    color: #dcdcdc;
}

QTreeWidget::item {
    border-bottom: 1px solid #555; /* Subtle separator for items */
}

QTreeWidget::item:selected {
    background-color: #5364A5; /* Blue for selected item */
    color: white;
}

QTreeWidget::item:hover {
    background-color: #43528B; /* Darker blue for hover item */
}

QTreeWidget::branch {
    background: #2b2b2b;
}

QTreeWidget::branch:has-siblings:!adjoins-item {
    border-image: url(path/to/your/line-image.png) 0; /* Custom image for branch lines */
}

QTreeWidget::branch:has-siblings:adjoins-item {
    border-image: url(path/to/your/branch-more-image.png) 0;
}

QTreeWidget::branch:!has-children:!has-siblings:adjoins-item {
    border-image: url(path/to/your/branch-end-image.png) 0;
}

/* Customize scroll bars */
QTreeWidget QScrollBar:vertical {
    border: 1px solid #2b2b2b;
    background: #2b2b2b;
    width: 10px;
    margin: 22px 0 22px 0;
}

QTreeWidget QScrollBar::handle:vertical {
    background: #5364A5; /* Blue for tree widget scrollbar handle */
    min-height: 20px;
}

QTreeWidget QScrollBar::add-line:vertical, QTreeWidget QScrollBar::sub-line:vertical {
    border: 1px solid #2b2b2b;
    background: #5364A5; /* Blue for tree widget scrollbar button */
    height: 20px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

/* Styles for the header (column titles) */
QHeaderView::section {
    background-color: #5364A5; /* Blue for header background */
    color: white;
    padding-left: 4px;
    border: 1px solid #5364A5; /* Blue for header border */
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTextEdit" name="info_bar">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:600;&quot;&gt;&amp;lt;Some data not sure what, maybe stats or soemthing&amp;gt; or fill me with buttons somewhere instead of the ... over there -&amp;gt;. Maybe do the CS and put them at the top in an action bar?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="options_button">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTreeView" name="client_tree_widget">
     <property name="styleSheet">
      <string notr="true">border-radius: 2px;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
